### PR TITLE
Created 8Knot Page

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

File: docs/Presentations/8Knot.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+ - Presentation
+ - Strategy
+list_image: /img/bok/page-types/presentation.png  
+---
+
+<link href="/css/presentation/web.css" rel="stylesheet"></link>
+
+<Bio name="Cali Dolfi" image="/img/people/cali-dolfi.jpeg" organisation="RedHat">
+Presentation by Cali Dolfi to FINOS OSR SIG on June 5 2024.
```
Comment (caradelia) [inline docs/Presentations/8Knot.md:15]: Cali Dolfi, Red Hat

**Review by caradelia — DISMISSED** [review]
Looks good. Just one small revision

**Comment by robmoffat** [comment]
hi @caradelia,

Can you update your review so I can merge?

thanks

**Comment by robmoffat** [comment]
@finos/osr-maintainers can we get this reviewed and merged?

**Review by psmulovics — APPROVED** [review]

**Pull request merged**